messages/src/msg_fields/protocols/coordinate_mediation/mediate_request.rs

```diff
@@ -0,0 +1,47 @@
+use serde::{Deserialize, Serialize};
+use typed_builder::TypedBuilder;
+
+use crate::{decorators::thread::Thread, msg_parts::MsgParts};
+
+pub type MediateRequest = MsgParts<MediateRequestContent, MediateRequestDecorators>;
```

Comment:
Given the nature of how these messages, I can't quite imagine practical scenario where the thread decorator would be used. I wouldn't mind keeping the second generic param at its default `NoDecorators` value. But not strongly opposing the current implementation either.

Reply:
I agree. The thread decorator does not seem very useful in request type messages. (Maybe relevant in response messages, quoting the request message `id`)
Using the default `NoDecorator` also makes message building slightly less verbose. 
In case some other Decorator fields seem relevant, we can later add them back.